### PR TITLE
Gxh 7 login page

### PR DESCRIPTION
**Pull request by Dubz97**

### CODE REVIEW

**Review by jakobmajiddd — APPROVED** [review]
Looks very cute!

File: templates/login.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    </section>
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+      integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+      crossorigin="anonymous"
+    ></script>
+  </body>
+</html>
+>>>>>>> d176f7ccb53ca70dcaac8f98ee50b9575f112060
```
Comment (Jarkyman) [inline templates/login.html:78]:
```suggestion
    </div>
</section>
</body>
</html>
```

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+            crossorigin="anonymous"></script>
+=======
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
+>>>>>>> d176f7ccb53ca70dcaac8f98ee50b9575f112060
```
Comment (Jarkyman) [inline templates/login.html:23]:
```suggestion    <!--    Helps us with styling, it's a framework helps us with mobile device formatting-->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
          integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
    <script defer src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
            integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
            crossorigin="anonymous"></script>
```

Fix merge conflikt

**Review by Jarkyman — CHANGES_REQUESTED** [review]
Merge conflikt :-P

**Review by Jarkyman — APPROVED** [review]

**Pull request merged**